### ISSUE TITLE
Projects's `settings.py` got shadowed by AREPL's `settings.py`

### ISSUE THREAD

**Issue by Ignisor** [Almenon/AREPL-vscode#314]
**Describe the bug**
In AREPL whenever I import anything that uses settings module in my project - I get an error `AttributeError: module 'settings' has no attribute 'SOME_NAME'`. Even in an empty project created to test that trouble. If I run the code simply using `python main.py` it works properly.

**To Reproduce**
1. Create an empty folder.
2. Create a virtual environment using Python 3.8.
3. Create a file named `settings.py`.
4. Add a variable there, for example, `TEST='Cucumber'`.
5. Create a file named `main.py` (or whatever.py).
6. Import settings and try to access the variable defined before:
    ```
    import settings

    if __name__ == "__main__":
         print(settings.TEST)
    ```
7. Execute the script using python in your environment: `python main.py`.
7. Activate AREPL and execute the file.
8. The error will be displayed on the top in the errors section of the AREPL.

**Expected behavior**
The code executes successfully without raising an exception.

**Screenshots**
<img width="1101" alt="image" src="https://user-images.githubusercontent.com/22328453/76747173-6586da00-6781-11ea-95bc-f3fa085a103c.png">

**Other Information (please complete the following information):**
 - OS: macOS
 - Python Version: 3.8

**Additional context**
In AREPL, if I execute `print(settings.__file__)` it shows `/Users/ignisor/.vscode/extensions/almenon.arepl-1.0.21/node_modules/arepl-backend/python/settings.py`.

**Comment by Almenon** [comment]
Crap. Thanks for the report, I'll try to have to fix out Tuesday.

**Comment by Almenon** [comment]
Sorry for the delay. This was surprisingly difficult. I investigated why arepl modules were being loaded instead of user modules and realized that the user's path was being inserted *after* the arepl path in sys.path, so when importing arepl files would be imported first. 

I fixed that but the problem still persisted... 🤔 

Then I realized that the imported modules were cached in sys.modules. I tried removing those, but that broke arepl_dump. arepl_dump imports from python_evaluator, which imports settings. Because I had just put the user path first in sys.path, python_evaluator used the user's `settings.py` file >_< .  I updated arepl_dump to temporarily put arepl first in sys.path so arepl imports would function correctly. At the end of arepl_dump it removes arepl from first ranking. 

Now everything is working again but I have a ugly bit code messing around with sys.modules. It would have been faster, simpler, cleaner, and more maintainable if I had just renamed everything....

I think I'll avoid the sunk cost fallacy and do just that. I'll save the code for removing modules in case I need it later.

**Comment by Almenon** [comment]
I just released 1.0.22. I named all my arepl files to start with arepl_ so they shouldn't conflict with your files anymore :)

**Comment by Ignisor** [comment]
Great! Thanks.

Just an idea. You may make a bash (or python, or whatever) script that renames files and changes imports, saves everything into a temporary folder, and then pushes it to the VSCode market. That way, you can keep your filenames clean and nice and avoid possible conflicts.
